packages/pds/src/app-view/api/app/bsky/util/postTypeSet.ts

```diff
@@ -0,0 +1,37 @@
+enum PostType {
```

Comment:
This file & class exist in two packages and perhaps that means it needs to be in one upstream package.